internal/terraform/module/module_ops.go

```diff
@@ -163,10 +164,10 @@ func ParseConfiguration(fs filesystem.Filesystem, modStore *state.ModuleStore, m
 		}
 
 		name := info.Name()
-		if !strings.HasSuffix(name, ".tf") || IsIgnoredFile(name) {
+		langID, err := lsp.LanguageIDFromPath(name)
```

Comment:
I understand the intentions, but as mentioned above - Language ID as a concept should ideally stay only on the top layer of LSP (handlers). I think there is no harm in still using `HasSuffix` and keep filtering by file extensions that way?